templates/20-04/aws.html

```diff
@@ -0,0 +1,232 @@
+
+  {% call(slot) vf_hero(
+    title_text='Ubuntu 20.04 LTS <br class="u-hide--small" />(Focal Fossa) on AWS',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
+      <p>
+        Ubuntu 20.04 LTS (Focal Fossa) is reaching the end of its 5 years of standard support on May 31, 2025, exposing your systems to unpatched vulnerabilities. To stay secure, you must act now. Explore your options below to ensure continued protection.
+      </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="/tutorials/how-to-upgrade-ubuntu-lts-to-ubuntu-pro-on-aws-using-aws-license-manager#1-overview"
+         class="p-button--positive">Upgrade now</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <p class="u-embedded-media">
+        <iframe title="How to perform in-place upgrade on AWS"
+                class="u-embedded-media__element"
+                src="https://www.youtube.com/embed/oYS28kmur2k"
+                allowfullscreen=""
+                frameborder="0"></iframe>
+      </p>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Standard support for Ubuntu 20.04 LTS ends on May 31, 2025</h2>
+      </div>
+      <div class="col">
+        <p>
+          Transitioning to an Ubuntu version with active standard support (Ubuntu 22.04 LTS or 24.04 LTS) is crucial for performance, hardware compatibility, and access to new technologies, making it ideal for new workloads. However, for existing deployments, the process may be more complex.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://ubuntu.com/about/release-cycle">Learn more about the Ubuntu lifecycle and release&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <hr class="p-rule" />
+      <div class="p-section--shallow">
+        <h2>What are your options?</h2>
+      </div>
+
+      <ol class="p-stepped-list--detailed">
+        <li class="p-stepped-list__item">
+          <h3 class="p-stepped-list__title">
+            Upgrade to an Ubuntu version with active standard support (Ubuntu 22.04&nbsp;LTS or 24.04&nbsp;LTS)
+          </h3>
+          <div class="p-stepped-list__content">
+            <h4 class="p-heading--5">For Ubuntu 22.04 LTS (coverage through 2027):</h4>
+            <hr class="p-rule--muted u-no-margin--bottom" />
+            <ul class="p-list--divided">
+              <li class="p-list__item has-bullet"><a href="/server/docs/upgrade-introduction">Upgrade directly from Ubuntu 20.04 LTS</a></li>
+              <li class="p-list__item has-bullet">
+                <a href="https://aws.amazon.com/marketplace/pp/prodview-dxdx74ozlxsl2">Launch a new instance of Ubuntu 22.04 LTS</a>
+              </li>
+            </ul>
+            <h4 class="p-heading--5">For Ubuntu 24.04 LTS (coverage through 2029):</h4>
```

Comment:
```suggestion
            <h4 class="p-heading--5">For Ubuntu 24.04 LTS (support through 2029):</h4>
```

This header is also missing hr above